bower.json

```diff
@@ -31,7 +31,8 @@
     "angular-vs-repeat": "~1.1.4",
     "medium-editor-tables": "~0.5.0",
     "exif-js": "^2.1.1",
-    "angular-i18n": "^1.5.3"
+    "angular-i18n": "^1.5.3",
+    "angular-history": "^0.8.1"
```

Comment:
If we do the above suggestion, then this can go away, and so can the addition to `karma.conf.js` below.